virttest/guest_agent.py

```diff
@@ -930,3 +930,14 @@ def get_users(self):
         cmd = "guest-get-users"
         self.check_has_command(cmd)
         return self.cmd(cmd=cmd)
+
+    def get_virtio_device(self):
+        """
+        Get virtio device driver info of windows guest.
+
+        :return: a list of virtio device driver info, such as device-id,
+                 driver-name, driver-version, driver-date and vender-id.
+        """
+        cmd = "guest-get-devices"
```

Comment:
Since this command is named `guest-get-devices`, does it only query virtio device or all type devices, I mean, if it is proper to name this function `get_virtio_device`?

Reply:
it only query virtio devices.
From upstream patch of qga.
add command guest-get-devices for reporting VirtIO devices